src/cogs/other/mobs.py

```diff
@@ -307,6 +309,8 @@ async def spawn_event(self, ctx):
 
                     ems_won = int((ems_won if ems_won > 0 else 1) * diff_multi)
 
+                    if await self.db.fetch_item(u.id, "Mooderald Sword") is not None:
+                        ems_won = int(ems_won * 1.1)
```

Comment:
add a newline after this if statement if it's not supposed to be part of the if/elif chain below it.

Reply:
oh oops